### ISSUE TITLE
Log in using Meteor Developer Account?

### ISSUE THREAD

**Issue by mizzao** [meteorpedia/wiki#8]
Instead of using the kitchen sink login with all possible types, why not get people to use their Meteor Developer Account to log in (accounts-meteor-developer)? Everyone who edits the wiki is probably going to have this account and it will link the users on the wiki to other developer services in the future.

**Comment by gadicc** [comment]
Hi All.  Uber apologies, for some reason my notifications were turned off for this repo!  In any event, as you noticed, the code here is based off a really old version of Meteor and is not actively maintained.

I've been working for some time on a complete rewrite from scratch, using all the latest Meteor practices, and introducing some super cool new features, some of which are almost done, but unfortunately, there is no ETA, as work priorities prevent me from actively working on this.  I could guess around June 2016 :>

To be clear though, the current site is still up and running and all content will be imported into the new version when it's ready.  I thank you all for your contributions and patience.

**Comment by gadicc** [comment]
Agree... Meteor logins didn't exist yet the last time I worked on this :>  Will of course be supported in v2.
